server/apps/archive/__init__.py

```diff
@@ -121,8 +121,12 @@ def init_app(app):
     superdesk.privilege(name='unspike', label='Un Spike', description='User can un-spike content.')
     superdesk.privilege(name='unlock', label='Unlock content', description='User can unlock content.')
     superdesk.privilege(name='metadata_uniquename', label='Edit Unique Name', description='User can edit unique name.')
-    superdesk.privilege(name='saved_searches', label='Manage Saved Searches',
-                        description='User can manage Saved Searches')
+    superdesk.privilege(name='global_saved_searches',
```

Comment:
can we refactor these privileges and saved_searches to it's own packages under user's or it's own?